conda_smithy/ci_register.py

```diff
@@ -15,20 +16,34 @@
     with open(os.path.expanduser('~/.conda-smithy/circle.token'), 'r') as fh:
         circle_token = fh.read().strip()
 except IOError:
-    print('No circle token. Put one in ~/.conda-smithy/circle.token')
+    print('No circle token.  Create a token at https://circleci.com/account/api and\n'
+          'put it in ~/.conda-smithy/circle.token')
 
 try:
     with open(os.path.expanduser('~/.conda-smithy/appveyor.token'), 'r') as fh:
         appveyor_token = fh.read().strip()
 except IOError:
-    print('No appveyor token. Put one in ~/.conda-smithy/appveyor.token')
+    print('No appveyor token. Create a token at https://ci.appveyor.com/api-token and\n'
+          'put it in ~/.conda-smithy/appveyor.token')
+
+try:
+    anaconda_token = os.environ['BINSTAR_TOKEN']
```

Comment:
I'm completely game for changing APIs. We are still in 0.x versions. I think it makes sense just to ditch this try..except altogether.